### DISCUSSION TITLE
사이드바에 보여지는 태그 갯수 지정 기능이 추가되었으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by CodeName393** [pronist/hELLO#95] [Q&A] [answered]
사이드바에서 태그가 너무 많아지면 보기가 힘들어질거같아 태그의 갯수를 지정하던가 아니면 태그가 밑으로가 아닌 옆으로도 같이 나열되어서 공간 활용을 했으면 좋겠습니다!
![캡처](https://user-images.githubusercontent.com/62503817/89746421-7c451900-daf4-11ea-95a4-5c97478866f7.PNG)

**Comment by pronist** [comment, marked as answer]
안녕하세요:)

해당 사항은 [티스토리 스킨가이드](https://tistory.github.io/document-tistory-skin/common/index.xml.html)에 의거하여
**index.xml** 에서 `tagsInCloud` 옵션을 설정하여 변경합니다. 별도로 스킨 옵션으로 만들지는 않을 예정입니다.

다만, 태그를 **여러 줄** 표기하는 것은 고려해볼 사항입니다.

```plain
tagsInCloud: 사이드바에 표시될 태그 갯수
```

그것에 더해 기본적으로 **티스토리 블로그 설정**에서 **사이드바 항목**에서 설정가능합니다.
